app/assets/javascripts/application/glossary.js

```diff
@@ -0,0 +1,31 @@
+$(document).on('mouseover', '.glossary-term, .glossary-definition', (e) => {
+  var def = $(e.target).closest('.glossary-term').find('.glossary-definition');
+  clearTimeout(def.data('timeout'));
+  def.data('timeout', null).show();
+});
+
+$(document).on('mouseout', '.glossary-term, .glossary-definition', (e) => {
+  var def = $(e.target).closest('.glossary-term').find('.glossary-definition');
+  if (!def.data('timeout')) {
+    def.data('timeout', setTimeout(() => {
+      def.data('timeout', null).hide();
+    }, 200));
+  }
+});
+
+$(document).on('click', '.glossary-term', (e) => {
```

Comment:
We need to consult Michelle/Soraya on what the UX should be on mobile. In the mean time can we just send the user directly to the term page rather than open the popup? We can use jQuery's `tap` event to differentiate between clicks and mobile interaction.

![glossary-mobile](https://user-images.githubusercontent.com/6469642/38059255-92674eec-329a-11e8-94dc-be603b5416cb.png)